### PR TITLE
fix: `Error` page layout

### PR DESCRIPTION
**Pull request by baseplate-admin**

### CODE REVIEW

**Comment by baseplate-admin** [comment]
The text needs a bit of resizing. See if @tokitou-san  can fix it



image

**Comment by tokitou-san** [comment]
> The text needs a bit of resizing.

Yes everything size looks off.
also @baseplate-admin what's your opinion about changing that fish's background gradient?
I like the gradient that we did in homepage

File: src/lib/components/pages/anime_info/error.svelte [hunk 1]
```diff
@@ -35,11 +35,13 @@
...
         transition:blur
-        class:md:flex-row-reverse={on_left}
-        class:md:flex-row={on_right}
-        class={cn(mapping.class, "relative flex h-full flex-col justify-end gap-20 md:items-end md:gap-0")}
+        class={cn(mapping.class, "relative grid h-full grid-flow-col justify-end gap-20 md:col-span-2 md:gap-0")}
     >
-        <error-context class="flex flex-col items-center leading-none md:mb-[13vw] md:w-[70dvw] md:items-start md:gap-[1vw] md:pl-[5vw]">
+        <error-context
+            class:order-1={on_right}
+            class:order-2={on_left}
```
Comment (baseplate-admin) [inline src/lib/components/pages/anime_info/error.svelte:42]: This is what you had to do

File: src/lib/components/pages/anime_info/error.svelte [hunk 2]
```diff
@@ -62,13 +64,12 @@
             </a>
         </error-context>
         <character-image
-            class="pointer-events-none relative flex items-end justify-center md:w-[45dvw]"
+            class:order-1={on_left}
+            class:order-2={on_right}
```
Comment (baseplate-admin) [inline src/lib/components/pages/anime_info/error.svelte:68]: This is what you had to do

**Comment by baseplate-admin** [comment]
image

This section needs a look

**Comment by tokitou-san** [comment]
Uhh does this looks good to you? well not for me :)
not this `grid-flow-col` add this `md:grid-flow-col`

Screenshot from 2023-09-16 12-52-49

File: src/lib/components/pages/anime_info/error.svelte (hunk 2 above)
Comment (tokitou-san) [reply src/lib/components/pages/anime_info/error.svelte:68]: Okay I did same, but I dont think we need this much `order` classes
just one add `order-1` for `on_right`
I mean no need this `order-` `order-2` ... on both `charater-image` and `error-context`

File: src/lib/components/pages/anime_info/error.svelte (hunk 1 above)
Comment (tokitou-san) [reply src/lib/components/pages/anime_info/error.svelte:42]: https://github.com/tokitou-san/CoreProject-Frontend/pull/760/files#r1327920187

File: src/lib/components/pages/anime_info/error.svelte (hunk 2 above)
Comment (baseplate-admin) [reply src/lib/components/pages/anime_info/error.svelte:68]: >I mean no need this order- order-2 ... on both charater-image and error-context

better to separate logics than to deal with snakes later

**Comment by baseplate-admin** [comment]
Why are you skipping deploy ?

File: src/lib/components/pages/anime_info/error.svelte [hunk 3]
```diff
@@ -35,12 +35,12 @@
...
     <section
         transition:blur
-        class={cn(mapping.class, "relative grid h-full justify-end md:grid-flow-col")}
+        class={cn(mapping.class, "grid grid-cols-12 h-full")}
     >
         <error-context
             class:md:order-1={on_right}
             class:md:order-2={on_left}
-            class="row-span-5 flex flex-col items-center self-end leading-none md:mb-[13vw] md:items-start md:gap-[1vw] md:pl-[5vw]"
+            class="row-span-6 flex flex-col items-center self-end leading-none md:mb-[13vw] md:items-start md:gap-[1vw] md:pl-[5vw] col-span-12 md:col-span-6"
```
Comment (baseplate-admin) [inline src/lib/components/pages/anime_info/error.svelte:43]: `col-span-12` ? Explain? what's it doing?

Comment (tokitou-san) [reply src/lib/components/pages/anime_info/error.svelte:43]: Heyo, I'm back, so uhh for small screen it gives full grid space
( doesnt work without this )

**Review by tokitou-san — APPROVED** [review]
LGTM

**Pull request merged**